server/scripts/import-tests/index.js

```diff
@@ -303,20 +303,20 @@ const updateJsons = async () => {
     const keysMjsPath = pathToFileURL(
         path.join(testsDirectory, 'resources', 'keys.mjs')
     );
-    const commands = Object.entries(await import(keysMjsPath)).map(
+    const commandsV1Parsed = Object.entries(await import(keysMjsPath)).map(
         ([id, text]) => ({ id, text })
     );
 
     // Write commands for v1 format
     await fse.writeFile(
-        path.resolve(__dirname, '../../resources/commands.json'),
-        JSON.stringify(commands, null, 4)
+        path.resolve(__dirname, '../../resources/commandsV1.json'),
+        JSON.stringify(commandsV1Parsed, null, 4)
     );
 
     try {
         // Commands path info for v2 format
         const commandsV2Path = pathToFileURL(
-            path.join(testsDirectory, 'commands.json')
+            path.join(testsDirectory, 'commandsV1.json')
```

Comment:
This would be `commandsV2.json`. But for this change to be valid, a PR on aria-at must also move [aria-at/tests/commands.json](https://github.com/w3c/aria-at/blob/master/tests/commands.json) to aria-at/tests/commandsV2.json, otherwise this path is invalid and subsequent imports would be wrong.